### ISSUE TITLE
Cemu crashes without a stack trace or any debugging related files

### ISSUE THREAD

**Issue by hYdos** [cemu-project/Cemu#1223]
### Current Behavior

Cemu crashes without printing a stack trace

### Expected Behavior

A stack trace should be printed in log.txt

### Steps to Reproduce

Grab the latest release of Cemu
Crash a game
Observe the log.txt

### System Info (Optional)

OS: Windows 10
GPU: RTX 2070 Super

### Logs (Optional)

[log.txt](https://github.com/cemu-project/Cemu/files/15449544/log.txt)

Homebrew I am working on which is crashing (it will be missing files so it won't crash the same way mine does, but it has the same missing stacktrace behavior)
[ReflectionMetaDumperWiiU.zip](https://github.com/cemu-project/Cemu/files/15449545/ReflectionMetaDumperWiiU.zip)

**Comment by Exzap** [comment]
Proper PPC stack traces can only be generated if Cemu is running in interpreter mode (by setting game profile CPU mode to interpreter or launch Cemu with `--force-interpreter`). In recompiler mode stack traces can not always be generated due to the way stack is managed. #641 will fix this partially

**Comment by hYdos** [comment]
The stack trace i'm looking at is one for Cemu, not the application running. Is there a way to get one for the application or is that out of scope for Cemu?

**Comment by Exzap** [comment]
What I said applies. It should print two stack traces, one for Cemu and another one for the PPC application running within it (if it is the PPC code that causes the crash)

**Comment by hYdos** [comment]
that is not true sadly, looking at logs you get a bunch of PPC info and the Cemu Stack trace.
[log.txt](https://github.com/user-attachments/files/16450709/log.txt)

**Comment by hYdos** [comment]
Unfortunately the latest version of Cemu crashes skylanders superchargers
at the title screen.

**Comment by hYdos** [comment]
Cemu 2.1 has fixed SSC for **some** people and I happen to be one of the lucky ones. Confirmed that i get a stack trace now. However, it only works with --force-interpreter and the ui options dont' work

**Comment by hYdos** [comment]
I wanted to update this again, Turns out Cemu does **not** generate a PPC stack trace when it should be all the time. It has consistently not worked the last 5 times but before that it was fine? I am able to do this using my physical WiiU but i use Cemu as a convenience tool.

**Comment by Exzap** [comment]
Cemu will print a PPC stack trace if the crash actually happened during the execution of game code. If it crashes within Cemu's code  it will print `Not active` under Active PPC instance but it should at least still print a list of all PPC threads. The relevant code is [here](https://github.com/cemu-project/Cemu/blob/0d8fd7c0dc2ea6ff750f4ef5b193ebc6388d31d0/src/Common/ExceptionHandler/ExceptionHandler.cpp#L75)

It would also really help if you actually posted logs and told us what you are trying to do
